libraries/AP_Scripting/lua_bindings.cpp

```diff
@@ -65,6 +65,285 @@ static const luaL_Reg servo_functions[] =
     {NULL, NULL}
 };
 
+// location stuff
```

Comment:
This is a bad rebase, this is all gone code.

Reply:
oups